moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -45,9 +45,12 @@
 #include <boost/bind.hpp>
 #include <limits>
 #include <memory>
-
+#include <typeinfo>
 namespace kinematic_constraints
 {
+
+rclcpp::Logger logger_kinematic_constraints = rclcpp::get_logger("kinematic_constraints");
```

Comment:
Ideally this would be a static const class variable. At minimum please find-replace logger_kinematic_constraints with LOGGER_KINEMATIC_CONSTRAINTS